server/base/src/main/java/org/apache/accumulo/server/manager/state/TabletManagementParameters.java

```diff
@@ -205,7 +206,7 @@ private static class JsonData {
 
     final Map<String,Set<String>> tserverGroups;
 
-    final Map<Long,Map<String,String>> compactionHints;
+    final Map<FateId,Map<String,String>> compactionHints;
```

Comment:
Making the key type for this map a string that is the canonical value would yield nicer and more stable json.

Reply:
If this seems like a big change, could add an ELASTICITY_TODO for this to get this PR merged.